<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>Prim</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Ordenar Distancia</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Grafo</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Ordenar Velocidad</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>20</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Origen x:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>80</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Destino x:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>80</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>110</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Velocidad:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>110</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>24</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>RED:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>140</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>140</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>GREEN:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>140</y>
            <width>54</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>140</y>
            <width>28</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>BLUE:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_9">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>140</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>40</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Origen y:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>40</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>80</y>
            <width>49</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Destino y:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>80</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Mostrar</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>R.Anchura</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>Dibujar Original</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>R.Profundidad</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>Dibujar Prim</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" rowspan="5" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusds">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionAbrir_2"/>
   </widget>
   <widget class="QMenu" name="menuParticula">
    <property name="title">
     <string>Particula</string>
    </property>
    <addaction name="actionMostar"/>
    <addaction name="actionPuntos_Cercanos"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmo">
    <property name="title">
     <string>Algoritmo</string>
    </property>
    <addaction name="actionkrustal"/>
   </widget>
   <addaction name="menusds"/>
   <addaction name="menuParticula"/>
   <addaction name="menuAlgoritmo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbrir_2">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionMostar">
   <property name="text">
    <string>Mostar</string>
   </property>
  </action>
  <action name="actionPuntos_Cercanos">
   <property name="text">
    <string>Puntos Cercanos</string>
   </property>
  </action>
  <action name="actionkrustal">
   <property name="text">
    <string>krustkal</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
